content/conference/2021_online_coding_sprint.md

```diff
@@ -10,19 +10,35 @@ homepage = "https://discuss.haiku-os.org/t/haiku-coding-sprint-2021/11327"
 
-The event is open to everyone who wants to work on Haiku or any related software
-project: write new applications, port existing applications, update already
-ported applications, no matter if it's your own or someone else's. And of course
-it's also possible to dig deeper into Haiku internals and fix some bugs there.
+## How does the coding sprint work?
+In past years, coding sprints involved gathering developers in a room (a meeting room, usually) where they would work on Haiku. Obviously, with this virtual coding sprint everything will run online, presumably with an IRC channel or another online platform.  
 
-The event overlaps with Hacktoberfest, so you can also take part into that if
-you want to.
+Anyone can participate, and there is **no obligation on how much time you put in** even though the sprint runs for two weeks. It's a good opportunity for existing contributors to work on something Haiku-related, like fix bugs or embark on that large project that's been sitting on their to-do list for a while, or help resurrect an app (might be their own or an app from the BeOS days sitting around in [HaikuArchives waiting for some TLC](https://github.com/HaikuArchives)), help finalise patches waiting on [Gerrit](https://review.haiku-os.org), submit proposals, etc. etc. And since there'll be more people online during this sprint, you'll have plenty of help from your fellow developers on hand!
+
+## What about new contributors?
+This is also a good chance for new contributors to get started too! Just to ensure you're ready for the sprint, it's a good idea to first check out our [page on how you can get started on development](https://www.haiku-os.org/community/getting-involved/developing), with ideas on what you might like to work on. You might also want to:
+* [Read the development FAQ](https://www.haiku-os.org/development/faq)
+* [Read the Step by Step Guide for new contributors](https://www.haiku-os.org/development/getting-started)
+* [Take a look at our coding guidelines](https://www.haiku-os.org/development/coding-guidelines)
+* [Read the Programming with Haiku series of lessons](https://www.haiku-os.org/development/programming_with_haiku)
+* [Join the development mailing list to stay up to date with the latest formal discussions ](https://www.freelists.org/list/haiku-development)
+* [Have a look around and join the forums](https://discuss.haiku-os.org) - [there's a topic on the coding sprint](https://discuss.haiku-os.org/t/haiku-coding-sprint-2021/11327) where you can post what you plan to work on.
+* Join the Haiku IRC/Matrix channel at `#haiku` (IRC) or `#haiku:matrix.org` (Matrix) where you can talk with other developers and Haiku enthusiasts, ask questions, get feedback and advice. Here, a response to any query is never too far away! Feel free to say hello and don't be afraid to ask if you have any questions.  
```

Comment:
If  you want to list bridges, please list all of them (including XMPP and BeShare). There is no reason to do free promotion for Matrix here over the other protocols. Also, you can't just refer to the IRC channel as "#haiku", you need to specify on which server it is, otherwise people won't find it.

Anyway, this will be a lot of useless information here, even after listing all the bridges, people will still have no idea how to connect to these things. So instead, link to the community/irc page, which can go in more length about the details of connecting to the channel. This way, people not familiar with IRC will find all the info they need there.